articles/container-registry/container-registry-tutorial-sign-trusted-ca.md

```diff
@@ -315,6 +315,9 @@ To learn more about assigning policy to a principal, see [Assign Access Policy](
         └── sha256:d7258166ca820f5ab7190247663464f2dcb149df4d1b6c4943dcaac59157de8e
     ```
 
+> [!NOTE]
+ > Notation uses [OCI Referrers Tag Schema](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#referrers-tag-schema) to store the signature in ACR by default. You can also enable [OCI Referrers API](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#listing-referrers) by using the flag `--force-referrers-tag false` if needed. The OCI Referrers API is supported by most of the ACR features except the CMK-encrypted registry.
```

Comment:
It is different default behavior for Notation v1 and Notation v2.

Reply:
As `notation` v2.0 is still in alpha, I believe we will update this doc until v2.0 is stable